pom.xml

```diff
@@ -31,7 +31,7 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.0</version>
                 <configuration>
-                    <source>1.8</source>
+					<source>1.8</source>
```

Comment:
Why is there so much space?

Reply:
Messed up formatting it seems, not sure. Any ideas for the freestanding signs?

1.14 works well so far, since i'm running it on 1.14 and the very long deprecated getData() methods are now finally removed. So it needs to be changed for the future.